src/Shared/SemanticConventions.cs

```diff
@@ -53,7 +53,7 @@ internal static class SemanticConventions
     public const string AttributeDbStatement = "db.statement";
     public const string AttributeDbOperation = "db.operation";
     public const string AttributeDbInstance = "db.instance";
-    public const string AttributeDbUrl = "db.url";
+    public const string AttributeDbUrl = "url.full";
```

Comment:
This attribute is used only by Elasticsearch package. It should be fine to rename it, but please replace also the C# const name. Without this, you are breaking convention used in the whole file.

```suggestion
    public const string AttributeUrlFull = "url.full";
```